forms/Form.php

```diff
@@ -371,6 +371,16 @@ public function httpSubmission($request) {
 				// Clear invalid token on refresh
 				$data = $this->getData();
 				unset($data[$securityID]);
+
+				// Ensure we don't send passwords back in clear-text
+				/** @var FormField $f */
+				foreach($this->Fields()->dataFields() as $f) {
+
+					if(get_class($f) === 'PasswordField' && isset($data[$f->getName()])) {
```

Comment:
.`$f instatnceof PasswordField`

You don't need to check `isset` for `unset` calls.

Also, can we name `$f` as `$field`, please?